.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
This appears to be specific to Grafana ? Can you please make it more generic and maybe add reference to the APIs being used.

Reply:
@dinogun Should I add some generic steps, or should I leave it at Steps to reproduce the behavior?

Reply:
`Steps to reproduce` should be good enough

Reply:
Made the change